woocommerce-myparcel.php

```diff
@@ -310,6 +310,10 @@ protected function upgrade($installed_version)
                 if (version_compare($installed_version, '4.2.1', '<=')) {
                     require_once('migration/wcmpbe-upgrade-migration-v4-2-1.php');
                 }
+
+                if (version_compare($installed_version, '4.3.6', '<')) {
```

Comment:
Kun je dit met `<=` doen zoals de rest?

Reply:
Dat kan helaas niet, want dan wordt de migratie bijna altijd 2 keer uitgevoerd (1 keer naar deze versie toe, een tweede keer van deze versie af naar de volgende), samen met Richard (want ik geloofde mijn ogen weer eens niet) vast kunnen stellen.

Reply:
Of we moeten een hogere versie gaan noemen, maar dat is denk ik verwarrender dan dit. Volgens mij is wat er nu staat ook daadwerkelijk wat er gebeurt, dus dat lijkt me op zich logisch. Staat alleen niet zo netjes met die voorgaande erbij, ik zou dan eerder de neiging hebben om die voorgaande ook in < te veranderen, alleen dan zijn er wel weer een hoop test werkzaamheden die erbij komen... denk ik.